tensorboard/plugins/image/images_demo.py

```diff
@@ -85,13 +86,13 @@ def convolve(image, pixel_filter, channels=3, name=None):
     Returns:
       A 3D `float32` `Tensor` of the same shape as the input.
     """
-    with tf.name_scope(name, "convolve"):
+    with tf.name_scope(name or "convolve"):
```

Comment:
wouldn't it be more correct to use `tf.compat.v1.name_scope`?

While at it, would it be possible to do `import tensorflowl.compat.v1 as tf` instead?

Reply:
Updated (requires removing existing compat.v1 calls)